code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -1028,6 +1028,292 @@ var/global/list/obj/item/device/pda/PDAs = list()
 							dat += {"<li>[mkr.name] ([mkr.x]/[mkr.y]) <a href='byond://?src=\ref[src];choice=removeMarker;rMark=[mkr.num]'>remove</a></li>"}
 						dat += {"</ul>"}
 
+			if (105)//Snake II

```

Comment:
Do you know how many modes there are on the PDA? I'm not making defines for every single of them when I'm only adding 3 new modes. Furthermore, I made apps modes as simple as I could (101,102,103, etc) unlike those cartridges modes. Curses to whoever coded the PDA cartridges.